### PR TITLE
fix: escape the regexp pattern to support the build metadata section …

### PR DESCRIPTION
**Pull request by skyf0cker**
…of version string

Fix issue https://github.com/outsideris/citizen/issues/122 to have supports for build section of provider version string

### CODE REVIEW

**Comment by skyf0cker** [comment]
@outsideris Hope for your review

File: lib/provider/cli.js [hunk 1]
```diff
@@ -78,7 +79,7 @@ module.exports = async (namespace, type, version, protocols, cmd) => {
     spinner.start();
 
     const files = await readdir(targetDir);
-    const providerRegex = new RegExp(`${fileNamePrefix}_(?<os>.*)_(?<arch>.*).zip`);
+    const providerRegex = new RegExp(`${escapeRegExp(fileNamePrefix)}_(?<os>.*)_(?<arch>.*).zip`);
```
Comment (outsideris) [inline lib/provider/cli.js:82]: we should escape `.` of `.zip`.

**Review by outsideris — COMMENTED** [review]
You are right. 
I leave a trivial comment.

File: lib/provider/cli.js (hunk 1 above)
Comment (skyf0cker) [reply lib/provider/cli.js]: Yep, have escaped it also.

**Review by outsideris — APPROVED** [review]
Thank you.

**Comment by outsideris** [comment]
Ignore CI results. It seems not problem by this commit.

**Pull request merged**